imagesloaded.js

```diff
@@ -349,7 +349,7 @@ Background.prototype.confirm = function( isLoaded, message ) {
 
 ImagesLoaded.makeJQueryPlugin = function( jQuery ) {
   jQuery = jQuery || window.jQuery;
-  if ( !jQuery ) {
+  if ( !jQuery || !jQuery.fn ) {
```

Comment:
Thanks for this contribution. Could you clarify how this resolves the bug? This code reads "if no jQuery OR no jQuery.fn." What circumstances lead to `.makeJQueryPlugin()` being called with an object that wasn't jQuery?

I'm not able to reproduce the vanilla JS bug with imagesLoaded.